crates/codegen/language/definition/src/model/terminals/trivia.rs

```diff
@@ -10,7 +10,6 @@ pub enum TriviaParser {
     Choice { parsers: Vec<TriviaParser> },
 
     Optional { parser: Box<TriviaParser> },
-    // TODO(#638): Remove this, once we adapt the DSL v1 codegen model to the new v2 definition.
```

Comment:
Anything that will touch the trivia as part of the #1020 will probably have to adapt this but doesn't strictly have to, so there's no point in keeping this TODO as it's optional and tracked elsewhere.

Moreover, we need to keep using `OneOrMore` for the current valid definition of leading_trivia and ZeroOrMore is unused but might be as part of #1020; I'm happy to remove it as part of this PR per YAGNI.